README.md

```diff
@@ -105,21 +105,45 @@ and one very simple Flask app.
 ### Setup
 
 - Recommended: create and activate a python 3.6 virtualenv:
-```
+```sh
 python3.6 -m venv pemm
 source pem/bin/activate
```

Comment:
I noticed another typo, this should be `pemm` instead of `pem`

Reply:
Good catch! I'll throw this fix in another branch

Reply:
@kmcelwee It's fine to just correct something this small in `develop` after you merge!

Reply:
Oh right! I was thinking of PUL, which has formal permissions that wouldn't allow something simple like this